devel/nlohmann-json-schema-validator/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        pboettch json-schema-validator 2.3.0
+github.tarball_from archive
+revision            0
+name                nlohmann-json-schema-validator
```

Comment:
why bot just name this as upstream calls it? What has the "nlohmann" to do with it except for the fact that this package depends on a port with that name?

Reply:
Upstream calls it in such a way https://github.com/pboettch/json-schema-validator/blob/813eb6312b6f323c4b75861d5972ec3d14b142c3/CMakeLists.txt#L17. Just for consistency. But I'm fine to rename it to json-schema-validator.